cmd/immuadmin/command/database.go

```diff
@@ -81,19 +81,26 @@ func (cl *commandline) database(cmd *cobra.Command) {
 				c.PrintfColorW(cmd.OutOrStdout(), c.Yellow, "Replication is a work-in-progress feature. Not ready for production use\n")
 			}
 
+			excludeCommitTime, err := cmd.Flags().GetBool("exclude-commit-time")
+			if err != nil {
+				return err
+			}
+
 			if err := cl.immuClient.CreateDatabase(cl.context, &schema.DatabaseSettings{
-				DatabaseName: args[0],
-				Replica:      isReplica,
+				DatabaseName:      args[0],
+				Replica:           isReplica,
+				ExcludeCommitTime: excludeCommitTime,
 			}); err != nil {
 				return err
 			}
 
 			fmt.Fprintf(cmd.OutOrStdout(), "database '%s' (replica = %v) successfully created\n", args[0], isReplica)
 			return nil
 		},
-		Args: cobra.ExactArgs(1),
+		Args: cobra.MaximumNArgs(2),
```

Comment:
Not sure I understand why this is needed - something changed in the number of positional arguments?